irohad/ordering/OrderingService.hpp

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef IROHA_ORDERINGSERVICE_HPP
+#define IROHA_ORDERINGSERVICE_HPP
+
+#include <spdlog/spdlog.h>
+#include <consensus/messages.hpp>
+#include <queue>
+#include <uvw.hpp>
+
+namespace ordering {
```

Comment:
In my opinion,
Generally namespace is same as directory/file name in irohad
`consensus/sumeragi.hpp` => `namespace consensus::sumeragi`;
if filename is same as directoryname, we abbreviates.
`foo/foo.hpp` => `namespace foo`;

In libs, first namespace is `iroha`
`libs/logger/logger.hpp` => `namespace iroha::logger`;

what do you think about?